src/json-patch-duplex.ts

```diff
@@ -677,4 +678,8 @@ if (typeof exports !== "undefined") {
   exports.validate = jsonpatch.validate;
   exports.validator = jsonpatch.validator;
   exports.JsonPatchError = jsonpatch.JsonPatchError;
-}
+
+  /* TS Transpiler automatically adds
+  .default  when referencing */
+  exports.default = jsonpatch;
```

Comment:
If you mean `export default module jsonpatch {...` then no, it throws an error. if you mean 
`export default  jsonpatch` in the end of the module you get `[ts] An export assignment cannot be used in a namespace.`

Is there a certain problem with this one? `export default jsonpatch` can't be inside an `if`, and `exports.default = jsonpatch` is transpiled to exactly the same thing in the js.